packages/api/src/car.js

```diff
@@ -122,7 +124,9 @@ export async function handleCarUpload (request, env, ctx, car, uploadType = 'Car
 
   await Promise.all([
     putToS3(env, s3Key, carBytes, carCid, sourceCid, structure),
-    putToR2(env, r2Key, carBytes, carCid, sourceCid, structure)
+    putToR2(env, r2Key, carBytes, carCid, sourceCid, structure),
+    writeSatNavIndex(env, carCid, carBytes),
+    writeDudeWhereIndex(env, sourceCid, carCid)
```

Comment:
idle thought: a shared AbortController so we bail on all if any has failed... or maybe just write the indexes only if the CAR upload has succeeded... I can't think of anything that gets worse if we end up with DUDEWHERE links and car indexes but no CAR, other than we could have failed on the first read (no entry in DUDEWHERE) rather than second read.